.travis/travis_install.sh

```diff
@@ -9,6 +9,6 @@ else
 fi
 
 source activate pyenv
-if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install --yes -q mkl numpy=1.9.1 scipy=0.14.0 nose=1.3.0 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx=1.5.1 mkl-service libgfortran=1 graphviz; fi
-if [[ $TRAVIS_PYTHON_VERSION == '3.4' ]]; then conda install --yes -q mkl numpy=1.9.1 scipy=0.14.0 nose=1.3.4 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx=1.5.1 mkl-service libgfortran=1 graphviz; fi
+if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then conda install --yes -q mkl numpy=1.13.0 scipy=0.19.1 nose=1.3.0 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx=1.5.1 mkl-service libgfortran=1 graphviz; fi
+if [[ $TRAVIS_PYTHON_VERSION == '3.4' ]]; then conda install --yes -q mkl numpy=1.13.0 scipy=0.19.1 nose=1.3.4 pip flake8=2.3 six=1.9.0 pep8=1.6.2 pyflakes=0.8.1 sphinx=1.5.1 mkl-service libgfortran=1 graphviz; fi
```

Comment:
Scipy 0.19.1 and numpy 1.13 have been out for little more than a year.  This not a lot of time for clusters and other slow-moving environments to update.  I would be hesitant to bump up the minimum versions like this.

Reply:
You're right, this is a really minor change to Theano and it shouldn't imply a bump up to a new required version of numpy. However, I think it would be nice to have the CI run the tests with different versions of numpy. Based on the errors I was getting when I tried to make travis install numpy 1.13.0, it seems that I will have to configure the tests for python3.6 also, or maybe 3.7 as @twiecki .

Reply:
Agreed, we should add a Python 3.7 + new numpy + new scipy configuration to travis in addition to the old ones, rather than replacing. That way we make sure minimum requirements as well as the most recent versions are supported.